text/0083-func-config-standard.md

```diff
@@ -0,0 +1,66 @@
+- **TEP**: [0](https://github.com/ton-blockchain/TEPs/pull/0)
+- **title**: FunC project configuration file
+- **status**: Draft
+- **type**: Meta
+- **authors**: [Danila Volkov](https://github.com/dvlkv), [Andrey Pfau](https://github.com/andreypfau)
+- **created**: 15.08.2022
+
+# Summary
+
+I propose to make a FunC project config, like project.json, Cargo.toml, etc.
+
+# Motivation
+
+Extensions, compilers and other dev tools need universal project configuration.
+
+# Guide
+
+Extensions can use it for optimal indexing source code, compilers can be configured without bash scripts.  
+Also, here comes more convinient project bootstapping. 
+
+# Specification
+
+I suppose it should be yaml file, named like .func.yml or something else.
```

Comment:
I think we need one unique name `ton.yml` without any other options. `func.yml` also ok, but I prefer `ton.yml`, it's really cool :)

Since most file systems hide files started with `.` I suggest not to use it.

Reply:
I believe that `func` in name is not optimal, since it will be better if one config format will be able to handle projects on different languages: func, tact, fift-asm, etc. 
`ton` is strange as well, why not `project`?

Reply:
We can collect more variants and make a poll ¯\_(ツ)_/¯

Reply:
This is more @andreypfau idea about naming config like `project` confusing while you have 100500 projects on different languages, so the name must be more ton-related. 

I got idea about diffrent languages, Didn't think of that at first.

Reply:
`contract` or `contracts` maybe?

Reply:
ton-project.yml?

Reply:
actor? I believe it will be better to migrate away from contract term in the long-term future.